remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyDecoder.java

```diff
@@ -45,6 +46,14 @@ public Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception {
             }
             return RemotingCommand.decode(frame);
         } catch (Exception e) {
+            in.resetReaderIndex();
```

Comment:
Features should not rely on malformed frames.  This is the hot data path, and is likely to impact performance too much.

Reply:
Under normal circumstances, there will not be many malformed frames. So this feature has almost no impact on performance. If there are a lot of malformed frames, it may be a packet flood attack. We should ban attacker ip.

@zhouxinyu do you have any better suggestions?

Reply:
> Under normal circumstances, there will not be many malformed frames. So this feature has almost no impact on performance. If there are a lot of malformed frames, it may be a packet flood attack. We should ban attacker ip.
> 
> @zhouxinyu do you have any better suggestions?

How about use RMQ admin CL?  @ShadowySpirits

Reply:
> > Under normal circumstances, there will not be many malformed frames. So this feature has almost no impact on performance. If there are a lot of malformed frames, it may be a packet flood attack. We should ban attacker ip.
> > @zhouxinyu do you have any better suggestions?
> 
> How about use RMQ admin CL? @ShadowySpirits

This tool is designed for application developers who use rocketmq sdk. They may not have experience using mqadmin, even if their runtime doesn't have jdk. So I hope to be able to use linux built-in software to achieve this function. @mxsm